src/Serilog/Configuration/LoggerDestructuringConfiguration.cs

```diff
@@ -28,28 +28,28 @@ public class LoggerDestructuringConfiguration
         readonly Action<Type> _addScalar;
         readonly Action<IDestructuringPolicy> _addPolicy;
         readonly Action<int> _setMaximumDepth;
-        readonly Action<int> _setMaximumStringLength;
-        readonly Action<int> _setMaximumCollectionLength;
+        readonly Action<int> _setMaximumStringCount;
```

Comment:
Looks like a find/replace error here - should still be `_setMaximumStringLength`.